tests/ScanCommandTest.php

```diff
@@ -96,7 +96,8 @@ protected function appearsInConsoleOutput($texts)
     }
 
     /** @test */
-    public function it_removes_output_file_on_overrite() {
+    public function it_removes_output_file_on_overrite()
```

Comment:
Please correct the spelling. Change `overrite` to `overwrite` everywhere in this package.